examples/cpp/pyperf/PyPerfBPFProgram.cc

```diff
@@ -289,6 +289,11 @@ on_event(struct pt_regs* ctx) {
   event->tid = (pid_t)pid_tgid;
   bpf_get_current_comm(&event->comm, sizeof(event->comm));
 
+  // Initialize stack info
+  event->kernel_stack_id = kernel_stacks.get_stackid(ctx, BPF_F_REUSE_STACKID);
+  event->stack_status = STACK_STATUS_ERROR;
```

Comment:
Worth to initialize `event->error_code = ERROR_NONE` or `ERROR_UNSET`? To catch any case were we submit an event without properly initializing the error code (thus, using the previous one...)

Reply:
There's no flow that allows that, but sure.

Reply:
Protecting from future bugs